common/misc.go

```diff
@@ -1,15 +1,23 @@
 package common
 
 import (
+	"errors"
+	"fmt"
 	"os/exec"
 
 	log "github.com/Sirupsen/logrus"
 )
 
 func HardResetHNS() error {
+	log.Infoln("Resetting HNS")
+	log.Debugln("Removing NAT")
+	if err := exec.Command("powershell", "Get-NetNat", "|",
+		"Remove-NetNat").Run(); err != nil {
+		log.Debugln("Could not remove container network.")
```

Comment:
no, I want to keep going with cleanup procedure. If Get-ContainerNetwork or Get-NetNat etc fails, maybe removing hns database will work.